docs/sources/k6/next/reference/versioning-and-stability-guarantees.md

```diff
@@ -0,0 +1,284 @@
+
+If the issue or bug also affects the supported version of k6, we apply the fix to it and publish a [patch version](#patch-version) of the currently supported version of k6.
+
+If the issue also affects a deprecated version of k6, we take no action and invite users to upgrade to at least the latest *supported version* of k6.
+
+#### An issue is found in the supported version of k6
+
+When a security issue or bug is found in the currently **supported** version of k6, the k6 team creates a fix in the support version’s branch. It publishes a new [patch version](#patch-version) of the supported version, which becomes the new supported version of k6.
+
+If the issue also affects the k6 version, which is currently the [major version](#major-version), a separate fix should be released as a [patch version](#patch-version) within the maintained version’s major scope.
+
+If the issue also affects deprecated versions of k6, we take no action and invite users to upgrade to at least the latest *supported version* of k6.
+
+#### An issue is found in a deprecated version of k6
+
+If an issue (security-related or a bug) is found in a **deprecated** version, we do not take action and recommend that users upgrade to a supported version of k6.
+
+## Experimental
+
+Within k6, some APIs, features, and modules are explicitly marked as experimental.
+
+Those APIs, features, and modules are either made of changes or new capabilities, which we believe are valuable to our users and the development of k6. Still, they are **not yet stable**.
+
+An experimental API, feature, or module **can undergo a breaking change and be completely removed without prior notice**. Our support policy does not cover experimental APIs, features, and modules. As such, we advise products, projects, and teams not to rely on them explicitly for mission-critical operations and to build upon them if possible.
+
+Experimental APIs, features, and modules may eventually become stable. However, they also may not.
+
+They should be marked and communicated internally and, most importantly, externally. When a user interacts with an experimental feature or module, it **must** be explicitly communicated to them. How to do it depends on the layer where the change is introduced and can vary case by case.
+
+For example, a JavaScript module should be recognizable as an experimental module by its import path, which includes the `k6/experimental/` prefix. Advertising the experimental state via the official documentation is also highly encouraged. However, using it as a unique level shouldn’t be considered sufficient communication.
+However, the development team is responsible to decide which is the reasonable level to apply based on the specific case.
+
+When an experimental API, feature, or module becomes stable, it will lead to a new minor version of k6 in most cases. The experimental module import path becomes deprecated and will be removed in a couple of minor releases with the future notice.
```

Comment:
```suggestion
Experimental APIs, features, and modules may eventually become stable. When that happens, it will lead to a new minor version of k6 in most cases. The experimental module import path becomes deprecated and will be removed in a couple of minor releases with a future notice.
```

Reply:
I think we can get rid of "However, they may also not."  portion of the sentence "Experimental APIs, features, and modules may eventually become stable. However, they also may not." that was a few lines above.